ddr/ddr/Models/GameEntities/Oncomer.swift

```diff
@@ -10,14 +25,25 @@ class Oncomer: SKSpriteNode, Spawnable {
   var passEffects: [Effect]
   var goneEffects: [Effect]
   
+  var collisionThird: Int
```

Comment:
If you are storing the third of the screen that the oncomer is on, it should be of type OncomerPosition